todo_app/templates/index.html

```diff
@@ -33,9 +68,7 @@ <h2>Items</h2>
             <td><a href="/delete?id={{item.id}}"> Delete </a></td>
           </tr>
           {%endfor%}
-        </table>
-        
-      </ul>
+        </ul> 
     </div>
   </div>
-{% endblock %}
\ No newline at end of file
+{% endblock %}§
```

Comment:
I'm guessing this character has snuck in somewhere!
```suggestion
{% endblock %}
```